### ISSUE TITLE
Does bigdl support  something like tf.nn.softmax_cross_entropy_with_logits？

### ISSUE THREAD

**Issue by land1725** [intel-analytics/BigDL#2886]
Hello,I am using bigdl to optimizes a multi-class multi-classification .But I only found MultiLabelMarginCriterion and MultiLabelSoftMarginCriterion to use.
Does bigdl support  something like tf.nn.softmax_cross_entropy_with_logits？
means  to creates a criterion that optimizes a multi-class multi-classification with softmax.

**Comment by zhangxiaoli73** [comment]
@land1725 
You can use `CrossEntropyCriterion`, details:
https://github.com/intel-analytics/BigDL/blob/master/spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/CrossEntropyCriterion.scala

**Comment by land1725** [comment]
CrossEntropyCriterion ：This criterion combines LogSoftMax and ClassNLLCriterion in one single class.

这个貌似不支持 多分类，多标签


------------------ 原始邮件 ------------------
发件人: "zhangxiaoli73"<notifications@github.com>;
发送时间: 2019年8月20日(星期二) 下午4:33
收件人: "intel-analytics/BigDL"<BigDL@noreply.github.com>;
抄送: "21809593"<land1725@vip.qq.com>;"Mention"<mention@noreply.github.com>;
主题: Re: [intel-analytics/BigDL] Does bigdl support  something like tf.nn.softmax_cross_entropy_with_logits？ (#2886)




@land1725
 You can use CrossEntropyCriterion, details:
 https://github.com/intel-analytics/BigDL/blob/master/spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/CrossEntropyCriterion.scala

**Comment by zhangxiaoli73** [comment]
yeah, we have not supported softmax_cross_entropy_with_logits with multi label, but you can try MultiLabelSoftMarginCriterion , it likes  sigmoid_cross_entropy_with_logits.
Hope it can help you. @land1725
